roles/nova-common/templates/etc/nova/policy.json

```diff
@@ -1,5 +1,5 @@
 {
-    "context_is_admin":  "role:admin",
+    "context_is_admin":  "role:admin or role:cloud_admin",
     "admin_or_owner":  "is_admin:True or project_id:%(project_id)s",
```

Comment:
Okay. I agree. However in my testing it looks like we need to specifically need to add the cloud_admin role to the admin_or_owner rule, so that the cloud admin is able to do all the operations that the admin is able to do on instances. is_admin:True is not inheriting from the context_is_admin.